pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -25,14 +24,16 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/netclone"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	"github.com/lf-edge/eve/pkg/pillar/utils"
+	fileutils "github.com/lf-edge/eve/pkg/pillar/utils/file"
 	"github.com/lf-edge/eve/pkg/pillar/vault"
 	"github.com/lf-edge/eve/pkg/pillar/zedcloud"
 	"github.com/shirou/gopsutil/host"
 )
 
 var (
-	nilIPInfo = ipinfo.IPInfo{}
-	smartData = types.NewSmartDataWithDefaults()
+	nilIPInfo   = ipinfo.IPInfo{}
+	smartData   = types.NewSmartDataWithDefaults()
+	maxReadSize = 65536 // From files in /persist
```

Comment:
If we use a different name in nodeagent it makes sense to use the same name here.